### ISSUE TITLE
[BUG] Window of launched apps are behind others

### ISSUE THREAD

**Issue by huafu** [lmenezes/cerebro#313]
### description

I'm using Cerebro on latest macOS with dual-screen (well, 3 exactly). It shows as expected on the main screen, but any application it launches will appear behind all other applications windows.

<img width="996" alt="screen shot 2018-07-16 at 07 18 23" src="https://user-images.githubusercontent.com/1162602/42743992-4865ecce-88c9-11e8-8a63-91263919afcf.png">

<img width="918" alt="screen shot 2018-07-16 at 07 18 38" src="https://user-images.githubusercontent.com/1162602/42744005-6934b62e-88c9-11e8-8955-53f8a0891cca.png">

### expectations

The application I'm launching should show its window on top of any existing window.

### system information

```yaml
  System:
    OS: macOS High Sierra 10.13.6
    CPU: x64 Intel(R) Core(TM) i5-4288U CPU @ 2.60GHz
    Memory: 287.36 MB / 8.00 GB
    Shell: 5.3 - /bin/zsh
  Binaries:
    Node: 10.4.0 - ~/.nvm/versions/node/v10.4.0/bin/node
    Yarn: 1.7.0 - /usr/local/bin/yarn
    npm: 6.1.0 - ~/.nvm/versions/node/v10.4.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Utilities:
    Make: 3.81 - /usr/bin/make
    GCC: 4.2.1 - /usr/bin/gcc
    Git: 2.15.2 - /usr/bin/git
  Servers:
    Apache: 2.4.33 - /usr/sbin/apachectl
  Virtualization:
    Docker: 18.03.1 - /usr/local/bin/docker
  Languages:
    Bash: 3.2.57 - /bin/bash
    Go: 1.10.3 - /usr/local/bin/go
    Java: 1.8.0 - /usr/bin/javac
    Perl: 5.18.2 - /usr/bin/perl
    PHP: 7.1.16 - /usr/bin/php
    Python: 2.7.15 - /usr/local/bin/python
    Ruby: 2.3.7 - /usr/bin/ruby

Cerebro: 0.3.1
```

Also I'm using `[CMD] + [SPACE]` as my launching shortcut, if it can make any difference...

**Comment by huafu** [comment]
Sounds like this has been fixed in `0.3.2`! Thanks!

**Comment by huafu** [comment]
whew, I was totally on the wrong repo, sorry, too much hours of geeking lately :-/
